### PR TITLE
fix(about-modal): makes changes for Openshift POC

### PR DESCRIPTION
**Pull request by matthewcarleton**
This is work requested by @kybaker for the Openshift POC.

The about modal has been simplified a bit by removing the secondary logo.

Also a bunch of minor design changes :)

### CODE REVIEW

File: src/patternfly/components/AboutModalBox/about-modal-box.scss [hunk 1]
```diff
@@ -118,33 +121,31 @@
...
-    "logo logo";
+    "content content";
 
   // Modal is full screen until larger breakpoint
   width: 100vw;
   height: 100vh;
   overflow-x: hidden;
   overflow-y: auto;
+  /* stylelint-disable */
+  scrollbar-color: #c10;
```
Comment (mcoker) [inline src/patternfly/components/AboutModalBox/about-modal-box.scss]: 🔥🔥🔥

should this be a var tho?

File: src/patternfly/components/AboutModalBox/about-modal-box.scss [hunk 2]
```diff
@@ -118,33 +121,31 @@
...
   background-color: var(--pf-c-about-modal-box--BackgroundColor); // Because this component is always dark, set the background color
   box-shadow: var(--pf-c-about-modal-box--BoxShadow);
+  grid-template-columns: var(--pf-c-about-modal-box--grid-template-columns);
```
Comment (mcoker) [inline src/patternfly/components/AboutModalBox/about-modal-box.scss]: this var isn't defined. should we define a default or just remove it? if we don't need it, I would probably just define `grid-template-columns` in the breakpoints

File: src/patternfly/components/AboutModalBox/about-modal-box.scss [hunk 3]
```diff
@@ -118,33 +121,31 @@
...
 
   @media only screen and (min-width: $pf-global--breakpoint--sm) {
-    grid-template-columns: 5fr 1fr;
+    --pf-c-about-modal-box--grid-template-columns: var(--pf-c-about-modal-box--lg--grid-template-columns);
```
Comment (mcoker) [inline src/patternfly/components/AboutModalBox/about-modal-box.scss]: should this be `var(--pf-c-about-modal-box--sm--grid-template-columns)`?

Comment (matthewcarleton) [reply src/patternfly/components/AboutModalBox/about-modal-box.scss]: 🤦‍♂️

File: src/patternfly/components/AboutModalBox/about-modal-box.scss (hunk 1 above)
Comment (matthewcarleton) [reply src/patternfly/components/AboutModalBox/about-modal-box.scss]: bah, I don't want that just playing around with it. Forgot to take it out. It is cool though, if It worked haha.

**Comment by mcoker** [comment]
Can you remove these vars? 

https://github.com/patternfly/patternfly-next/blob/034a935e4bec8b0968c61ffb443aea42037a95af/src/patternfly/components/AboutModalBox/about-modal-box.scss#L84-L99

**Pull request merged**